services/src/kilda-core/kilda-model/src/main/java/org/openkilda/model/Switch.java

```diff
@@ -68,6 +68,7 @@
 
     private String description;
 
+    @Property(name = "under_maintenance")
```

Comment:
should't this part be in the previous commit?

Reply:
Yes, you are right. I forgot to write this annotation in the previous commit. But we plan to merge this branch in develop as soon as possible and tests will slow it down. Anyway, these commits need to be merged together.

Reply:
Fixed.